test/unit/sites.js

```diff
@@ -0,0 +1,181 @@
+'use strict';
+
+const chai = require('chai');
+const assert = chai.assert;
+const sites = require('../../routes/sites');
+
+describe('utils', function() {
+  describe('sites', function() {
+    it('gets all sites', function(done) {
+      var fakeData1 = {
```

Comment:
It may be easier to define some reusable global fake data variables up top to that you don't have to define them every time (and changing them is easier that way). This is just a suggestion though.